src/styles.scss

```diff
@@ -5,13 +5,24 @@
 @import "./variables.scss";
 @import "./app/style-components/wp-block-columns.scss";
 @import "./app/style-components/pricing-table.scss";
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 400;
+  src: url('./assets/SourceSansPro-Regular.woff2') format('woff2'), /* Замените путь на соответствующий файл шрифта */
+       url('./assets/SourceSansPro-Regular.woff') format('woff'); /* Замените путь на соответствующий файл шрифта */
+}
```

Comment:
The font-faces for the other fonts (semibold, light etc.) are missing but font files are there.